src/framework/preferences/css/stylus/SeparatedPanelPrefsEditor.styl

```diff
@@ -63,6 +62,8 @@
     }
 
     .fl-prefsEditor-showHide {
+        background-color: #fff;
+        color: #2f2b2a;
```

Comment:
should these be using the variables? The styling for the reset button makes use of the variables.

Reply:
Maybe, I just stopped at recreating the stylus logic from here with custom properties: https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/css/stylus/utils/Themes.styl#L90-L116

Reply:
Resolved in 289c3f6d1.